app/assets/stylesheets/application.css.scss

```diff
@@ -0,0 +1,132 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import url('https://fonts.googleapis.com/css?family=Lobster|Open Sans|');
+
+body {
+  margin-left: 20px;
+  margin-right: 0 auto;
+}
+
+btn.title_link {
+  padding: 10px 16px;
+  border-radius: 6px;
+}
+
+div.title_link {
+  display: inline-block;
+  font-family: Lobster;
+  padding: 10px 16px;
+  font-size: 23px;
+  line-height: 3;
+  border-radius: 6px;
+}
+
+div.title_link_holder {
+  display: flex;
+  justify-content: center;
+  padding-top: 10px;
+  padding-bottom: 30px;
+}
+
+.font_18 table {
+  font-size: 18px;
+}
+
+h1.title_page {
+  font-family: Lobster;
+  font-size: 2cm;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+h1, h2, h3, h4, h5, h6 {
+      text-align: center;
+  }
+
+header.cows {
+  margin: 0 auto;
+}
+
+header img.cows {
+  width: 100%;
+  height: 30%;
+  margin: 0 auto;
+  min-width: 1200px;
+  min-height: 200px;
+}
+
+#home_market_table {
+  max-width: 80%;
+}
+
+#market_link {
+  padding: 10px 20px;
+  font-size: 23px;
+  line-height: 3;
+  border-radius: 6px;
+}
+
+.narrower_table a {
+  color: grey;
+}
+
+#vendor_link {
+  padding: 10px 20px;
+  font-size: 23px;
+  line-height: 3;
+  border-radius: 6px;
+}
+
+.market_info {
+  text-align: center;
+}
+
+.market_info div {
+  display: inline-block;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.narrower_table {
+  margin: 0 auto;
+  max-width: 100%;
+  margin-bottom: 20px;
+  padding: 8px;
+  line-height: 1.428571429;
+  vertical-align: top;
+  border-top: 1px solid #ddd;
+  display: table;
+  border-collapse: separate;
+  border-spacing: 2px;
+  border-color: grey;
+}
+
+.narrower_table td {
+  min-width: 200px;
+}
+
+nav {
```

Comment:
You can assign `p` and `nav` the same CSS style by separating them with the comma:
`p, nav {` etc